### PR TITLE
GRW-1551/ NO Travel form template

### PR DESCRIPTION
**Pull request by amandaflod**
## Describe your changes

Add NO travel form template. 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1551]: https://hedvig.atlassian.net/browse/GRW-1551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/services/PriceForm/data/NO_TRAVEL.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Template } from '../PriceForm.types'
+
+export const NO_TRAVEL: Template = {
+  name: 'NO_TRAVEL',
+  initialData: {
+    isStudent: false,
+  },
```
Comment (robinandeer) [inline apps/store/src/services/PriceForm/data/NO_TRAVEL.ts:7]: For this case -- perhaps there's a point in giving users the option to let us know they are students? Let's ask Zak maybe?

**Review by robinandeer — APPROVED** [review]
Good job!

Hehe only one question per section -- perhaps we can just put it under one. But good for now.

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/services/PriceForm/data/NO_TRAVEL.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/services/PriceForm/data/NO_TRAVEL.ts:7]: What does it mean?  We don't sell travel insurance to students in Norway or is it something else?

**Review by alebedev — APPROVED** [review]

File: apps/store/src/services/PriceForm/data/NO_TRAVEL.ts (hunk 1 above)
Comment (amandaflod) [reply apps/store/src/services/PriceForm/data/NO_TRAVEL.ts:7]: Oh I will check with Zak! :)

**Pull request merged**